### DISCUSSION TITLE
Page number in header doesn't update

### DISCUSSION THREAD

**Discussion by DeMarcoAlberto** [typst/typst#3770] [Q&A]
Hi, I wrote the following for the header of my thesis but the page number is updated only since the second page. I would like to set the first page number to 3, than 4, 5... Instead I get 1, 4, 5...
If I write the same code for footer instead of header the page number works like I want. Thank you very much to whoever helps me!

#set page(header: context [Chapter 1 - Formula SAE #h(1fr) #counter(page).display("1")] + line(length: 100%))
#counter(page).update(3)

**Comment by istudyatuni** [comment]
Try move `#counter(page).update(3)` before `#set page`

**Comment by DeMarcoAlberto** [comment]
If I do this the whole header disappears
